src/web/reducers/flows.js

```diff
@@ -81,6 +84,18 @@ function createSnapshot(state) {
   }
 }
 
+function findNodesThatReferenceFlow(state, flowName) {
+  let nodes = []
```

Comment:
This variable (reference) is never changed, so can and should be `const`. Same for `flow` and `node` below. We should probably add am eslint rule for this, I find it very helpful to have consts everywhere when possible (except for about 5% of cases when mutations are really practical)

Reply:
Agreed, nice spotting!